assets/pages/regular-contributions/components/contributionsCheckoutContainer.jsx

```diff
@@ -2,6 +2,7 @@
 
 // ----- Imports ----- //
 
+import React from 'react';
```

Comment:
The same question as above, I thought that the container only knows about the state and it is not its responsibility to know about presentational logic.